docs/atsd_install.md

```diff
@@ -0,0 +1,162 @@
+# Installing Axibase Time-Series Database (ATSD) as the backend for cAdvisor 
+
+[Axibase Time Series Database](http://axibase.com/products/axibase-time-series-database/) can collect Docker metrics through cAdvisor for long-term retention, analytics and visualization. A single ATSD instance can collect metrics from many Docker hosts and cAdvisors.
+
+In the standard setup cAdvisor monitors the Docker Host, ATSD container and itself. All statistics are sent over TCP protocol to ATSD. Other local containers will be automatically monitored by cAdvisor and their statistics will be sent to ATSD.
+
+Remote Docker hosts can be easily monitored with multiple cAdvisor instances sending data to a centralized ATSD server. ATSD will store metrics from local and remote Docker hosts for consolidated monitoring and analytics. This type of setup will allow for centralized capacity planning and performance monitoring.
+
+![Distributed Docker Infrastructure](http://axibase.com/wp-content/uploads/2015/01/docker_distributed.png)
+
+[Learn more about cAdvisor](http://axibase.com/products/axibase-time-series-database/writing-data/docker-cadvisor/)
+
+#### Quick Start: Running cAdvisor in a Docker Container with ATSD as the storage driver
+
+Start ATSD in new container:
+
+```
+sudo docker run \
```

Comment:
(nit) drop `sudo` here and below. If users need it they can add it, but we shouldn't encourage unnecessary sudo use.